### PR TITLE
[stable/graphite] Upgrading Graphite version to 1.1.4

### PR DESCRIPTION
**Pull request by deniszh**
1.1.4 is latest stable Graphite version, plus docker image 1.1.4-1 contains valuable fixes - see https://github.com/graphite-project/docker-graphite-statsd/releases/tag/v1.1.4-1

/cc @fabian-schlegel @monotek 

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@deniszh please sign off your commits, see: https://www.helm.sh/blog/helm-dco/index.html

File: stable/graphite/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-version: 0.1.1
-appVersion: "1.1.3"
+version: 0.1.2
+appVersion: "1.1.4-1"
```
Comment (davidkarlsen) [inline stable/graphite/Chart.yaml:3]: please update values.yaml and readme too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/graphite/Chart.yaml (hunk 1 above)
Comment (deniszh) [reply stable/graphite/Chart.yaml:3]: Done

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**